video/out/opengl/video.c

```diff
@@ -3391,7 +3393,8 @@ static void check_gl_features(struct gl_video *p)
     bool have_mglsl = gl->glsl_version >= 130; // modern GLSL (1st class arrays etc.)
     bool have_texrg = gl->mpgl_caps & MPGL_CAP_TEX_RG;
     bool have_tex16 = !gl->es || (gl->mpgl_caps & MPGL_CAP_EXT16);
-    bool have_compute = gl->glsl_version >= 430; // easiest way to ensure all
+    bool have_compute = gl->DispatchCompute;
```

Comment:
I guess, but I think most uses of compute shaders are going to end up using 2D arrays, especially all of the user shaders. In fact it pretty much seems to only be the HDR detection code so far that doesn't need them. I'd prefer simpler checks that are more readable and more maintainable over trying to get the exact minimum condition right. If even `mesa` supports array_of_array then I can't imagine it would make much of a difference.